### ISSUE TITLE
Implement MathLive Toolbar

### ISSUE THREAD

**Issue by alexrcabral** [benetech/MathShare#722]

**Comment by johnhbenetech** [comment]
Have you evaluated this for a11y? The current version on mathlive.io doesn't appear to be accessible in a few brief experiments.

I would really need a list of exactly what is needed to make this a compliant, agreeable solution for us:
* Should buttons be tab focusable?
* Should every button have on-hover info
* Should those tooltips include shortcut info?
* What should be read by a screen reader for each button?
* How would a keyboard-user navigate between buttons? How would they navigate between tabs?

As we evaluate all of the above and see how much work might be required to bridge those gaps, it might be a better approach to list what major features of the mathlive keyboard we do like and see how much work it would be to evolve our current palette in line with those requirements.
